client/blocks/edit-gravatar/index.jsx

```diff
@@ -187,6 +190,28 @@ export class EditGravatar extends Component {
 						{ isUploading && <Spinner className="edit-gravatar__spinner" /> }
 						</div>
 				</FilePicker>
+				<div>
+					<p className="edit-gravatar__explanation">Your profile photo is public.</p>
+					<InfoPopover position="left">
+						{ translate( '{{p}}The avatar you use on WordPress.com comes ' +
+							'from {{a}}Gravatar{{/a}} - a universal avatar service.{{/p}}' +
+							'{{p}}Your photo may be displayed on other sites where ' +
+							'you use your email address rockstar@email.com.{{/p}}',
+							{
+								components: {
+									a: <a
+										href={ gravatarLink }
+										target="_blank"
```

Comment:
we usually flag links that open in a new window with an "external" icon. see https://github.com/Automattic/wp-calypso/tree/master/client/components/external-link